### ISSUE TITLE
ftp

### ISSUE THREAD

**Issue by sacko1966** [QuickBox/QB#201]
After new install of latest quickbox free or community version supporting ubuntu18.04 ftp does not work and no matter how hard i have tried it just will not connect over ftp keep getting error could not connect to server error 530 login incorrect but the login is fine for rutorrent which is the same password

**Comment by JMSDOnline** [comment]
I'll post an update to this on the installer. This is more than likely due to the signed key and the latest RSA requirements.
Please run the following:
```bash
openssl req -config /root/.openssl.cnf -x509 -nodes -days 720 -newkey rsa:2048 -keyout /etc/ssl/private/vsftpd.pem -out /etc/ssl/private/vsftpd.pem
```
Then issue a restart to the vsftp service `service vsftpd restart`

**Comment by sacko1966** [comment]
I dont think it is that as i have tried this and still the same error ftp
for super user is fine
when adding a new user that is when the error occurs in filezilla but the
username and password are fine as it is the same login details for
rutorrent

Status: Connecting to 195.154.184.59:21...
Status: Connection established, waiting for welcome message...
Status: Initializing TLS...
Status: Verifying certificate...
Status: TLS connection established.
Command: USER andy
Response: 331 Please specify the password.
Command: PASS *******
Response: 530 Login incorrect.
Error: Critical error: Could not connect to server

**Comment by JMSDOnline** [comment]
Default port for TLS/SSL (SFTP) is `4747` and for additional created users is standard FTP on port `5757` as listed on the Readme. Are you connecting via those ports? As I see `21` listed in your error message.

**Comment by sacko1966** [comment]
Yes i have tried all ports 5757 21 ive disabled ufw i have checked configs
everything something lurking in ubuntu 18.04 it just will not connect for
some reason but it does for super user i even commented out root in ftp
user list to try it but same error

**Comment by sacko1966** [comment]
The problem seems to be with /usr/bin/lshell but this is to advanced for me
permissions and user files not found

**Comment by JMSDOnline** [comment]
Check that lshell is actually compiled with `which lshell`. 

I just went to check and the installer here is attempting to build lshell from apt, however, that apt source is no longer functional/existent. So if this is a recent install, that may be why. Even if it is somehow there, I'd suggest `apt -y autoremove lshell` and build it from a more current, albeit no longer maintained repo (which still works and gets the job done as it should). 

Seen here on the recent edits I just posted to the quickbox-setup build file:
https://github.com/QuickBox/QB/commit/bbed587860f12679fe7319445dae4bc5b847b1c3#diff-d3fd950c9a4caebd335d5a73a5670052c18349f27f27d8af9118f156667b9fe2L571-L574
